x-pack/plugins/maps/public/connected_components/mb_map/tooltip_control/tooltip_popover.test.tsx

```diff
@@ -66,7 +66,9 @@ const mockMBMap = ({
 const defaultProps = {
```

Comment:
Can you remove layerList and geoFields, they are no longer props for TooltipPopover